copyrighter/copyrighter.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python3
```

Comment:
the python linting checks that run on nimble should also run in this repo

Reply:
Yeah i'm working on setting that up now. I did check these file with the nimble linting before moving it over here

Reply:
I've added pre-commit to this PR. Github actions is not setup to call pre-commit automatically.

Reply:
Ok now github actions calls it automatically